test/test_filters.py

```diff
@@ -35,3 +36,22 @@ def test_get_element_by_tag_nested():
     """, 'div')
     print(result)
     eq_(result, """<div>foo</div><div>bar</div>""")
+
+
+def test_json_format_filter():
+    json_format_filter = JsonFormatFilter(None, None)
+    result = json_format_filter.filter(
+        """{"field1": {"f1.1": "value"},"field2": "value"}""")
+    print(result)
+    eq_(
```

Comment:
Also this could be written as:
```
    eq_(result, """{
    "field1": {
        "f1.1": "value"
    },
    "field2": "value"
}""")
```

Reply:
This is definitely easier format. I already failed with the oneline format when changing this test to have different indentation. With this multi-line format it is easier to see what I'm writing as the expected formatting.